perf/Makefile

```diff
@@ -1,9 +1,15 @@
-all: randbytes handshake sslnew newrawkey rsasign x509storeissuer providerdoall rwlocks pkeyread evp_fetch
+all: randbytes handshake sslnew newrawkey rsasign x509storeissuer providerdoall rwlocks pkeyread evp_fetch \
+	dh.pem dhx.pem dsa.pem ec.pem rsa.pem x25519.pem \
+	dh.der dhx.der dsa.der ec.der rsa.der x25519.der
```

Comment:
These are really confusing names for aliases of pkeyread. Would you please consider something like `pkeyread-dh-pem`?

Also on Windows this will not be practical IMO. So perhaps just keeping the key type and formats as arguments and handling this within the perftesting script somehow would make more sense.

Reply:
New version of diff uses suggested naming convention. The whole point of this exercise is to unify command line options handling of performance tools. In my opinion the effort here definitely pays off because  then we can construct performance testing scripts without too much thinking which arguments to pass to each tool. It shortens debugging time spent to get tests running.

My plan for windows is to use `COPY` instead of symlink.

Reply:
I am not sure about this approach. IMO it is not scalable. @openssl/otc what are others opinions?

Reply:
I think it could be made scalable using a make variable to hold the list of possibilities.
Perhaps using some string manipulation functions to get things correct and only have one copy of the list.